### ISSUE TITLE
Autocomplete onChange returns event, not value.

### ISSUE THREAD

**Issue by junibrosas** [react-toolbox/react-toolbox#986]
My Autocomplete component has this onChange prop `onChange={(value) => {console.log(value)}}`.  Before, it returned a value, now it returned an event object but the target property is null. The input text component is working, only the auto complete is not.

**Comment by zakdances** [comment]
I have the same issue. The `onChange` function doesn't get called with a value as documented, it gets called with an *`Event`* object and also a *`Proxy`* object.

```
const myAutocompleteOnChangeCallback = (arg1, arg2) => {
    console.log(arg1); // Event {isTrusted: false, type: "react-mousedown", target: react, currentTarget: react, eventPhase: 2…}
    console.log(arg2) // Proxy {dispatchConfig: Object, _targetInst: ReactDOMComponent, _dispatchInstances: ReactDOMComponent, nativeEvent: MouseEvent, type: "mousedown"…}
}
```

**Comment by thomasthiebaud** [comment]
I'm facing the same problem, do you know which version introduce this bug ? I thought it was `1.2.4` but I cannot make it work in `1.2.3` either.

**Comment by thomasthiebaud** [comment]
Is a version `1.2.6` with this fix (and others) is coming before `2.0.0` ?

**Comment by javivelasco** [comment]
Of course, I'm trying to merge stuff and close easy issues this week. As soon as I get it more or less ready I'll publish a new release with all of these fixes

**Comment by thomasthiebaud** [comment]
Thanks a lot ;)

**Comment by junibrosas** [comment]
The fix is included in the `1.3.0` release. Thanks @javivelasco @borela
